config/accounts.go

```diff
@@ -221,3 +226,23 @@ func (a *AccountIntegration) GetByIntegrationType(integrationType IntegrationTyp
 
 	return integrationEnabled
 }
+
+// AccountHooks represents account-specific hooks configuration
+type AccountHooks struct {
+	Modules       AccountModules    `mapstructure:"modules" json:"modules"`
+	ExecutionPlan HookExecutionPlan `mapstructure:"execution_plan" json:"execution_plan"`
+}
+
+// AccountModules mapping provides account-level module configuration
+// format: map[vendor_name]map[module_name]json.RawMessage
+type AccountModules map[string]map[string]json.RawMessage
+
+func (m AccountModules) ModuleConfig(id string) json.RawMessage {
+	ns := strings.SplitN(id, ".", 2)
+	if len(ns) < 2 {
+		glog.Errorf("Can't find module config, ID must consist of vendor and module names separated by dot")
```

Comment:
We may need to change this error handling so a single error doesn't end up flooding the logs. Let's look at it again once a caller to `ModuleConfig` is added.